### ISSUE TITLE
Modular Sources

### ISSUE THREAD

**Issue by Sadwhy** [fakeyatogod/AnimeScrap#68]
Built in sources are a big legal issue. Having sources built right into the app puts it into the red zone. With a high chance of getting a DMCA. Putting sources outside of the app(separate github repo) and making the extension manager like what miru has done would put the app into a legal gray zone. I suggest that you move from built in sources as soon as possible. Bonus if you integrate anilist into it so it can work under the guise of just a regular anilist tracker.

**Comment by fakeyatogod** [comment]
Yeah I have thought of that. Will require some effort and time. As I am not free at all times. Collaborations are welcome.
